src/main/java/dev/jbang/cli/BaseBuildCommand.java

```diff
@@ -329,8 +329,10 @@ static void createJarFile(ScriptSource src, RunContext ctx, File path, File outp
 			}
 		}
 
-		String runtimeOpts = Stream	.concat(src.getRuntimeOptions().stream(), ctx.getPersistentJvmArgs().stream())
-									.collect(Collectors.joining(" "));
+		// When persistent JVM args are set they override any runtime options set on the
```

Comment:
are they no longer persistent ?

Reply:
yes, but all runtime options are always persistent, so it wasn't really necessary to give them different names